### ISSUE TITLE
[Stout] Thinkpad x131e thinks it's suspended on reboot (no fan)

### ISSUE THREAD

**Issue by cablespaghetti** [MrChromebox/firmware#466]
Hi,

I've got a couple of these devices and have run various versions of your firmware and various linux distros since early last year. First of all, thanks for the work put into making this possible!

The issue I've got is that when powered off and on, or suspended the devices work as normal. However when rebooted the little red indicator light blinks like the device is suspended and critically the fan does not cut in causing overheating issues. The workaround is to never reboot, only suspend or power off/on.

However I've recently put one into service as a laptop for my son, so it's being used more heavily and made me more keen to figure out how to fix this.

I'm pretty technical but know almost nothing about firmware. Let me know anything I can do to help you debug this or figure it out myself.

Thanks again!

I believe these might be related to the same issue but I'm not certain. Also I'm not sure anyone has noticed the light blinking like the device is suspended:
https://github.com/MrChromebox/firmware/issues/131
https://www.reddit.com/r/chrultrabook/comments/b4ouqn/lenovo_x131e_chromebook_stout_fan_issues/

**Comment by MrChromebox** [comment]
my x131e is no longer functional so I have no way to test this device currently. I'm not even sure what would cause these issues for this device only, since the other Sandybridge/Ivybridge platform devices do not have similar issues.

**Comment by cablespaghetti** [comment]
Fair enough. It is super weird! I think it’s time I learn to debug firmware.

**Comment by MrChromebox** [comment]
the problem with STOUT is that it's using a closed-source, non-Google EC. That makes it very hard to debug things like the fan control as there is limited info available about the EC-AP interface

**Comment by cablespaghetti** [comment]
> the problem with STOUT is that it's using a closed-source, non-Google EC. That makes it very hard to debug things like the fan control as there is limited info available about the EC-AP interface

Thanks 2013 era Lenovo engineers! I'm convinced now from the blinky red light that whatever the firmware is doing for a reboot is making the EC think the device is in sleep. I suppose the debug process is going to be sending random signals to the EC and hoping for the best 😆

**Comment by cablespaghetti** [comment]
I thought I'd experiment and install a SeaBIOS ROM from here: https://johnlewis.ie/custom-chromebook-firmware/rom-download/

The problem does not exist for that ROM. I would obviously much prefer to have a UEFI ROM which somebody is supporting, but it is interesting. Now I get to figure out what they do differently on reboot.

**Comment by MrChromebox** [comment]
you're looking at probably 5-8 years of firmware changes, if not more, depending on what branch that image was built from. But we know it's EC related at least, so that narrows it down
